lib/std/math.zig

```diff
@@ -36,33 +36,38 @@ pub const sqrt2 = 1.414213562373095048801688724209698079;
 /// 1/sqrt(2)
 pub const sqrt1_2 = 0.707106781186547524400844362104849039;
 
-// From a small c++ [program using boost float128](https://github.com/winksaville/cpp_boost_float128)
-pub const f128_true_min = @bitCast(f128, @as(u128, 0x00000000000000000000000000000001));
-pub const f128_min = @bitCast(f128, @as(u128, 0x00010000000000000000000000000000));
-pub const f128_max = @bitCast(f128, @as(u128, 0x7FFEFFFFFFFFFFFFFFFFFFFFFFFFFFFF));
-pub const f128_epsilon = @bitCast(f128, @as(u128, 0x3F8F0000000000000000000000000000));
+pub const floatExponentBits = @import("math/float.zig").floatExponentBits;
+pub const floatMantissaBits = @import("math/float.zig").floatMantissaBits;
+pub const floatExponentMin = @import("math/float.zig").floatExponentMin;
+pub const floatExponentMax = @import("math/float.zig").floatExponentMax;
+pub const floatTrueMin = @import("math/float.zig").floatTrueMin;
+pub const floatMin = @import("math/float.zig").floatMin;
+pub const floatMax = @import("math/float.zig").floatMax;
+pub const floatEps = @import("math/float.zig").floatEps;
+
+// TODO These should eventually be removed, see #10133
+pub const f16_true_min: comptime_float = floatTrueMin(f16);
```

Comment:
Nit: add the values as comments behind, so they can be used for tests later.

Reply:
Do you mean adding the deprecated definitions, like this?

```zig
pub const f16_true_min: comptime_float = floatTrueMin(f16); // 0.000000059604644775390625
```

... or do you mean an equation that is equivalent to what the value is, like:

```zig
pub const f16_true_min: comptime_float = floatTrueMin(f16); // 2^-15
```

... or something else? Not sure what you mean.

Reply:
you can also add both, if you feel like it. Though the first should be sufficient.